CourseDevelopment/Week5/Chapter5.MD

```diff
@@ -0,0 +1,99 @@
+# Course Development Week 5
+Course Development Week 5 for the Open Science Course for physics students at graduate level. This document is based on the Syllabus.MD.
+
+Date: Dec/07/2022
+
+Name: Hendrik Snijder @hendelhendel
+
+Open tasks @SanliFaez @erikvansebille
++ Feedback on lecture-tutorial distribution of topics
++ Feedback on lecture format
++ Feedback on tutorial format
+
+## Goal: public engagement: story-telling and presentation; activism and trust in scientists
+After this week, one understands the role, responsibilities and interaction of scientists with scientific communities and the general public. 
+Students have to give a presentation about their other assignments using skills learned in the course. 
```

Comment:
How are we going to do this practically? Each student gives one presentation? Perhaps better to call it a 'pitch', so that it's short? A bit like https://www.uu.nl/en/research/breaking-science?

Also, can the students choose which other assignment? Or do you have a particular one in mind?

Reply:
Yes, I agree, lets call it a pitch. Later in the chapter it is called a like that as well.

Reply:
So let's change then to 'pitch'
```suggestion
Students have to give a pitch about their other assignments using skills learned in the course. 
```